rasa/nlu/train.py

```diff
@@ -79,7 +79,7 @@ async def train(
             training_data_endpoint, nlu_config.language
         )
     elif isinstance(data, TrainingDataImporter):
-        training_data = await data.get_nlu_data(nlu_config.data)
+        training_data = await data.get_nlu_data(nlu_config.language)
```

Comment:
@ricwo That's the bug fix which I referred to in the other comment

Reply:
👍